src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -564,11 +564,12 @@ private void markAsRead() {
         }
     }
 
-    private void createNewWatchListMenuItem() {
+    /**
+     * Creates a new watch list panel containing the currently selected issue
+     */
+    private void createNewWatchListPanel() {
         Optional<GuiElement> item = listView.getSelectedItem();
-        if (!item.isPresent()) {
-            return;
-        }
+        assert item.isPresent();
```

Comment:
Is it okay if we display a dialog to show the error? This way, it would become immediately apparent to the user that an error has occured and we can fail the current operation.